### ISSUE TITLE
bug: Serialization failed for object of type CompletionResponse in Llamaindex integration

### ISSUE THREAD

**Issue by jonas-loxo** [langfuse/langfuse#3861]
### Describe the bug

I'm using llamaindex and the integration errors when attempting to serialize. Here's a stacktrace:

```
Serialization failed for object of type CompletionResponse
Traceback (most recent call last):
  File "/Users/jonas/Library/Caches/pypoetry/virtualenvs/app/lib/python3.10/site-packages/langfuse/serializer.py", line 77, in default
    return obj.model_dump()
  File "/Users/jonas/Library/Caches/pypoetry/virtualenvs/app/lib/python3.10/site-packages/pydantic/main.py", line 390, in model_dump
    return self.__pydantic_serializer__.to_python(
TypeError: 'MockValSer' object cannot be converted to 'SchemaSerializer'
```

I added some logging to the langfuse serializer (wrapping [this](https://github.com/langfuse/langfuse-python/blob/17f1ff16c7efe23a3989c33752476f49cc664385/langfuse/serializer.py#L77) line in a try except block) and logging `obj.__dict__` when an exception triggers: 

```
{'text': 'False', 'additional_kwargs': {}, 'raw': ChatCompletion(id='...', choices=[Choice(finish_reason='stop', index=0, logprobs=None, message=ChatCompletionMessage(content='False', refusal=None, role='assistant', audio=None, function_call=None, tool_calls=None))], created=1729591581, model='gpt-4o-mini-2024-07-18', object='chat.completion', service_tier=None, system_fingerprint='...', usage=CompletionUsage(completion_tokens=1, prompt_tokens=1164, total_tokens=1165, completion_tokens_details=CompletionTokensDetails(audio_tokens=None, reasoning_tokens=0), prompt_tokens_details=PromptTokensDetails(audio_tokens=None, cached_tokens=1024))), 'logprobs': None, 'delta': None}
```

It appears to be that what's an ChatCompletion Response is treated as CompletionResponse

### To reproduce

Set up python 3.10 env, and I believe I used the llamaindex OpenAI Completion API. We're using async python, are streaming and are wrapping the llamaindex APIs with sync_to_async from asgiref.sync. Then simply run a single call.

```
llm = OpenAI(model="gpt-4o-mini", api_key=settings.OPENAI_API_KEY)
resp = await sync_to_async(llm.complete)("...")
```

### SDK and container versions

langfuse 2.52.1
llama-index-core = ">=0.11.0,<0.12.0"
llama-index-llms-openai = "^0.2.15"

### Are you interested to contribute a fix for this bug?

No

**Comment by maheshtambe** [comment]
+1
we are also facing same issue while tracing openai completion api using langfuse instrumentation module with llamaindex.

**Comment by hieu-cove** [comment]
I'm also observing the same issue on:
```
langfuse==2.52.2
llama-index==0.11.19
openai==1.52.2
pydantic==2.9.2
```

**Comment by El-Hassan-Hajbi** [comment]
Hi all, any updates on the issue ?

**Comment by hassiebp** [comment]
Thanks for raising this @jonas-loxo - we will take a look!

**Comment by hassiebp** [comment]
We have shipped a potential fix in v2.53.2 . Could you please upgrade and try again?
